etc/barbican_audit_map.yaml

```diff
@@ -0,0 +1,21 @@
+service_type: 'data/security'
+service_name: 'barbican'
+
+prefix: '/v1'
+
+resources:
+  secrets:
+    children:
+      metadata:
```

Comment:
there is no multiple instances of metadata per secret, right? So this should be a singleton. Otherwise the middleware would expect an object id.

metadata is also a good example on how *keys* are modelled